runtime/autoload/man.vim

```diff
@@ -70,12 +72,14 @@ endfunction
 function! man#read_page(ref) abort
   try
     let [sect, name] = man#extract_sect_and_name_ref(a:ref)
-    let [b:man_sect, name, path] = s:verify_exists(sect, name)
+    let [sect, name, path] = s:verify_exists(sect, name)
     let page = s:get_page(path)
   catch
-    " call to s:error() is unnecessary
+    " call to s:error() does nothing because we were
+    " called from the BufReadCmd autocmd.
```

Comment:
~~There was no error shown though. Try `tabe man://printsssdsds`. Even if you add a `s:error(v:exception)` call (though I'm not entirely sure why), no error is printed.~~

Aha, I had `shortmess+=F` which hides the message. So we should have the call to `s:error` there.

Do you think it's a bug that `shortmess+=F` is hiding messages from `:echo`?

Help says

```
don't give the file info when editing a file, like :silent
was used for the command
```

but perhaps it shouldn't just `:silent`, but instead, only not print the file info message?